### DISCUSSION TITLE
Draft Data Objects in Pimcore

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#15814] [Q&A / Support]
Without activating workflow in which table the draft data objects are stored, I'm trying to retrieve those data objects which is in draft mode, but can't to find out in which table that are stored in pimcore. Anybody have an idea about this kindly help me.
@fashxp @kingjia90

**Comment by SamyMP** [comment]
Depending on your [version storage config](https://pimcore.com/docs/platform/Pimcore/Tools_and_Features/Versioning#version-storage), it could be either on your FileSystem (which I believe is the default), or in your database. 

Either way, you can see the version in the `versions` table

**Comment by KasthuriAnandhan** [comment]
If workflow is not enabled how to retrieve the the data objects those are in draft by code.
